src/react/webpack.config.js

```diff
@@ -17,27 +20,30 @@ const intro = `/*!
  * Build: ${(new Date()).toISOString()}
  */`;
 
-import resolve from 'rollup-plugin-node-resolve';
-import commonjs from 'rollup-plugin-commonjs';
-import babel from 'rollup-plugin-babel';
-import replace from 'rollup-plugin-replace';
-
-export default {
-  entry: 'src/react/app.js',
-  dest: 'dist/react/app.js',
+module.exports = {
+  entry: './src/react/app',
+  output: {
+    path: './dist/react',
+    filename: 'app.js'
+  },
+  module: {
+    loaders: [{
+      test: /\.js$/,
+      exclude: /node_modules/,
+      loader: 'babel-loader',
+      query: {
+        presets: ['latest'],
```

Comment:
babel-preset-env acts the same as preset-latest when you don't use options if you want to use that.

and usage is similar the same: `presets: ['env']`. Going to add an uglify option later in https://github.com/babel/babel-preset-env/issues/134 otherwise can try babili.